jobs/config.json

```diff
@@ -948,6 +948,18 @@
   ]
 },
 
+"ci-kubernetes-e2e-charts-gce": {
+  "scenario": "kubernetes_e2e",
+  "args": [
+    "--env-file=platforms/gce.env",
+    "--env-file=jobs/ci-kubernetes-e2e-charts-gce.env",
+    "--test=false",
```

Comment:
I don't think I have privileges, can someone help me push a new image? I'm fixing the test issue now, but it should be all set after that. The file `charts/test/helm-test-e2e.sh` does not exist and the test will fail at first. Once I setup the tests in the charts repo, the tests should start running. Does that sound reasonable?